### PR TITLE
Flat Styling

### PR DESCRIPTION
**Pull request by delphiactual**
#968
https://github.com/DestinyItemManager/DIM/compare/dev...chrisburgin95:dev

Pulled all the changes closing #968 in favor of this

### CODE REVIEW

**Comment by delphiactual** [comment]
@chrisburgin95

**Comment by bhollis** [comment]
Give me a chance to check out this branch and poke around at the changes.

**Comment by delphiactual** [comment]
This actually does break some things
 - [ ] Can not hide categories currently
 - [ ] Weird fadeaway on itemdetails when closed

**Comment by SunburnedGoose** [comment]
Some screenshots would be helpful.

**Comment by delphiactual** [comment]
screenshot 2017-02-22 16 22 19

screenshot 2017-02-22 16 22 58

**Comment by bhollis** [comment]
It might be nice to break this up into several changes - for example the load outs menu and headers look cool, while I'm not so sure about the dialogs. 

-Ben

**Comment by bhollis** [comment]
Before:
screen shot 2017-02-25 at 4 10 48 pm

After:
screen shot 2017-02-25 at 4 12 58 pm

Notable:
* Removed line under header
* Removed "inset" effect on category headers
* Removed separation around loadout items

**Comment by bhollis** [comment]
(BTW, just as an aside, "flat design" is a few years out of date by now. These days, designers are all about rounded boxes again, and huge purple gradients.)

File: src/index.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <link href='https://accounts.google.com' rel='preconnect'>
     <link href='https://ssl.gstatic.com' rel='preconnect'>
 
-    <link href='https://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700' rel='stylesheet' type='text/css'>
```
Comment (kyleshay) [inline src/index.html]: we're only using 200, 400/normal and 700/bold, and no italic variants in our styles (aside the settings page, but that's only for bold.) if we pair this back down we'll see slightly faster load time for this typeface & its font-weights.

we should also see where we're using 200 and what it looks like compared to 300, as that is what we're pulling down.

**Pull request merged**